src/components/button/button.scss

```diff
@@ -31,6 +31,15 @@
     min-width: initial;
   }
 
+  &.btn-float {
+    box-shadow: var(--kd-elevation-level-3);
+  }
+  &.btn-hidden {
+    opacity: 0;
+    visibility: hidden;
+    transition: opacity 300ms ease-in-out, visibility 0ms ease-in-out 300ms;
```

Comment:
There's still 2 duration values on visibility, one of them might get treated a a delay value. Need to remove the 0ms.